go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/aws/amazon-cloudwatch-agent
 
-go 1.13
+go 1.17
 
-replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20201015165757-4470de2d306b
+replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20220502160831-c20ebe67c5ef
 
-replace github.com/shirou/gopsutil => github.com/aws/telegraf/patches/gopsutil v0.0.0-20201015165757-4470de2d306b // Temporary fix, pending PR https://github.com/shirou/gopsutil/pull/957
+// Temporary fix, pending PR https://github.com/shirou/gopsutil/pull/957
+replace github.com/shirou/gopsutil/v3 => github.com/aws/telegraf/patches/gopsutil/v3 v3.0.0-20220502160831-c20ebe67c5ef // indirect
```

Comment:
Is this going to be addressed or should we just expect this to stay here indefinitely

Reply:
For now there is no good solution going forward, we will just keep it up-to-date like telegraf, so this will be here going forward.